site/docs/v1/tech/apis/prometheus.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: doc
+title: Prometheus API
+description: API for retrieving application metrics to be used with prometheus
+---
+
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+
+== Overview
+
+This page contains the API that is used for retrieving FusionAuth application metrics to be used with Prometheus. Please refer to the link:/docs/v1/tech/tutorials/prometheus#[Prometheus setup] guide to understand how to setup Prometheus with the FusionAuth Prometheus endpoint.
+
+
+* <<Retrieve the Metrics>>
+
+
+== Retrieve the Metrics
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/prometheus/metrics#
+--
+
+=== Authentication
+Prometheus does not provide functionality to authenticate using custom HTTP headers. As a workaround, we are using API key as password using "basic auth" authorization method. In order to authenticate using basic auth please use string "apikey" as the username and a valid API key as the password. The API key must have `GET` permission for the `/api/prometheus/metrics` endpoint.
+
+Please refer to the link:/docs/v1/tech/apis/authentication#basic-authentication-with-api-key[Authentication] document for more details about `basic auth` authentication method using API keys.
```

Comment:
You can take a look at how we document the Status API for an example. We usually just list the same URI twice - each with a different link for the type of authentication. https://fusionauth.io/docs/v1/tech/apis/system/#retrieve-system-status

I would not describe what we are doing as a work-around, it is just an authentication method you can use that is compatible with Prometheus.  You can describe how to perform this here, or if you want to push it all into the authentication doc that is fine too.

Reply:
~Is the authentication doc for `/docs/v1/tech/apis/authentication#basic-authentication-with-api-key` under a different PR?~

Found it https://github.com/FusionAuth/fusionauth-site/pull/648/files